docs/HowTo/GetStarted/migration.md

```diff
@@ -75,3 +75,15 @@ A node running on GoQuorum 2.6.0 can coexist on a network where other nodes are
         To enable account unlocking explicitly use `--allow-insecure-unlock`:
 
         `--allow-insecure-unlock ` Allow insecure account unlocking when account-related RPCs are exposed by http.
+
+    * **Failure to sync up**
+      Quorum 2.6.0 can fail to sync up with `missing parent` error under the following scenario:
+      In a Quorum network running with gcmode=full and block height exceeding immutability threshold(with blocks in freezerdb), if a node is restarted non-gracefully(kill -9/docker kill & start) then it can fail to sync up with its peers with `missing parent` error.
+
+      This is due to an upstream bug where non-graceful restart causes  gap between leveldb and freezerdb.
```

Comment:
Indent this line by two more spaces for it to appear on the same bullet point as line 79.

Reply:
done